pkg/util/validation/validate.go

```diff
@@ -120,11 +121,21 @@ func NewValidateMetrics(r prometheus.Registerer) *ValidateMetrics {
 		[]string{"user"},
 	)
 	registerCollector(r, histogramSamplesReducedResolution)
+	labelSizeBytes := prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name:                            "cortex_label_size_bytes",
```

Comment:
Should we call it `cortex_series_label_size_bytes`?  `cortex_label_size_bytes` makes me confused that this metric is for a single label not combined label size total